ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/WorldUpdater.java

```diff
@@ -77,7 +77,7 @@ default MutableAccount getOrCreate(final Address address) {
    * @return the account {@code address} as modifiable object, or {@code null} if the account does
```

Comment:
some of the javadoc might need changing. The `DefaultEvmAccount` itself is not modifiable.